lua/terrortown/entities/items/item_ttt_armor.lua

```diff
@@ -19,6 +19,12 @@ function ITEM:Equip(buyer)
 	end
 end
 
+function ITEM:Reset(buyer)
+	if SERVER then
+		buyer:DecreaseArmor(GetConVar("ttt_item_armor_value"):GetInt())
```

Comment:
Hmmm... It's tricky. Is there a possibility to get a negative amount? What, if I bought another armor and this item removes (but I already got damage and lost armor)...

Reply:
Therefore the function "DecreaseArmor", it handles the capping

Reply:
That is not a valid solution, since it is possible to have many sources, where additional armor comes from, this would remove armor points, without considering the armor that is left, from this item. (eg. get 2 armor items +100, get shot -10, remove one -100 and you got less than you should